Xamarin.Forms.Platform.Android/Renderers/SearchHandlerAppearanceTracker.cs

```diff
@@ -41,6 +43,22 @@ public SearchHandlerAppearanceTracker(IShellSearchView searchView)
 			UpdateInputType();
 		}
 
+		void SearchHandlerFocusChangeRequested(object sender, VisualElement.FocusRequestArgs e)
+		{
+			e.Result = true;
+
+			if (e.Focus)
+			{
+				_control?.RequestFocus();
+				_control?.PostShowKeyboard();
```

Comment:
here is checked for null, but below not.
Maybe in the first line of the method add `if (_control == null) return;` ?